communication/slack-guidelines.md

```diff
@@ -40,24 +40,46 @@ administrators have time. There is no explicit interval.
 
 [Slack Archive Download]
 
-### DM (Direct Message) Conversations
-
-Please do not engage in proprietary company specific conversations in the
```

Comment:
FWIW I think it would be good to retain a reminder that the slack is logged and that we're not guaranteeing privacy for sensitive discussions that should've happened somewhere else / the purpose of the slack is to discuss the OSS project(s)

Reply:
See lines 56-57.

Reply:
[Thanks, I would mark this resolved if I could]